taichi/codegen/codegen_opengl.cpp

```diff
@@ -351,6 +352,7 @@ float atomicMin_ext_ns_f32(int addr, float rhs)
   return intBitsToFloat(old);
 }
 );
+      // clang-format on
```

Comment:
Is this the correct usage?

Reply:
See here:
https://github.com/taichi-dev/taichi/blob/62cdc60f2e3d981b4d5c1c9f60b7590c9e8031a1/taichi/backends/metal/shaders/runtime_kernels.metal.h#L42-L106

You'll need to turn it on/off 4 times :-)